### PR TITLE
DENG-3096 Add QA checks & split into separate DAGs to schedule at different times

### PR DESCRIPTION
**Pull request by kwindau**
Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-4082)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/cloudflare_derived/browser_usage_v1/checks.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#warn
+{{ is_unique(["dte", "device_type", "user_type", "location", "browser", "operating_system"]) }}
```
Comment (kik-kik) [inline sql/moz-fx-data-shared-prod/cloudflare_derived/browser_usage_v1/checks.sql:2]:
```suggestion
{{ is_unique(["date", "device_type", "user_type", "location", "browser", "operating_system"]) }}
```

I assume the field is called `date`?

**Review by kik-kik — COMMENTED** [review]
Out of curiosity, why are we doing these in separate DAGs?

File: dags.yaml [hunk 2]
```diff
@@ -1709,7 +1709,7 @@ bqetl_census_feed:
   tags:
     - impact/tier_2
 
-bqetl_cloudflare:
```
[the remark this answers is not in the document]
Comment (kwindau) [reply dags.yaml:1712]: Good to know, will do!

File: sql/moz-fx-data-shared-prod/cloudflare_derived/browser_usage_v1/checks.sql (hunk 1 above)
Comment (kwindau) [reply sql/moz-fx-data-shared-prod/cloudflare_derived/browser_usage_v1/checks.sql:2]: Yeah it's dte in all 3 tables

**Comment by kwindau** [comment]
> Out of curiosity, why are we doing these in separate DAGs?
Good question - basically there is a limit to how many I can times I can hit the API with my token in a set amount of time - because these are auto generated, it's detecting no dependencies and scheduling all 3 concurrently.  I thought about ordering them in a certain specific order instead, but my thought is I'd rather move them into separate DAGs so that I can schedule them at very different times throughout the day - so that's why each DAG is like totally at different times now.  This will also make my life easier for validating each one/marking it as complete, and then adding telemetry-airflow to each one once it's ready to be used (I don't want to spam everyone with failure emails while I'm still working out the kinks)

**Review by Marlene-M-Hirose — APPROVED** [review]

**Pull request merged**